src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs

```diff
@@ -218,6 +230,72 @@ private List<string> GetExamples(XPathNavigator nav, ITripleSlashCommentParserCo
             return GetMultipleExampleNodes(nav, "/member/example").ToList();
         }
 
+        private void ResolveCodeSource(XDocument doc, ITripleSlashCommentParserContext context)
+        {
+            foreach (XElement node in doc.XPathSelectElements("//code"))
+            {
+                var source = node.Attribute("source");
+                if (source == null)
+                {
+                    continue;
+                }
+
+                var region = node.Attribute("region");
+
+                var path = source.Value;
+                if (!Path.IsPathRooted(path))
+                {
+                    path = Path.Combine(context.Source?.BasePath ?? ".", path);
```

Comment:
Oh, and I realize ```Source?.BasePath ?? "."``` isn't 100% correct since "." is unlikely to be correct no matter where the intent is here. It's my understanding (could very well be wrong) that BasePath shouldn't ever be null when running in the actual tool, but it was always null in unit tests. I took a fast/cheap way out to ensure we didn't throw a NullReferenceException here, being fine with likely producing a warning message in this unexpected scenario. If that's not good enough, I can fix that however is deemed appropriate as well.